changes/3544.changed

```diff
@@ -0,0 +1,2 @@
+The default database backend if `METRICS_ENABLED` is `True` is now "django_prometheus.db.backends.postgresql"
+The default CACHES backend if `METRICS_ENABLED` is `True` is now "django_prometheus.cache.backends.redis.RedisCache"
```

Comment:
Since `development/nautobot_config.py` and `nautobot/core/tests/nautobot_config.py` both set METRICS_ENABLED=True, probably all of the `DB_ENGINE` settings in `.github/workflows/` and in `development/docker-compose.mysql.yml` need to be updated as well. Would you mind doing that as part of this PR?